internal/backends/sqlite/collection.go

```diff
@@ -74,7 +74,7 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 		if v != nil {
 			if id, ok := v.(types.ObjectID); ok {
 				whereClause = fmt.Sprintf(` WHERE %s = ?`, metadata.IDColumn)
-				args = []any{string(must.NotFail(sjson.MarshalSingleValue(id)))}
+				args = []any{must.NotFail(sjson.MarshalSingleValue(id))}
```

Comment:
`string()` just converted `[]byte` to `string` for better logging. It did nothing for quotes, I think.

Reply:
I think it should be `fmt.Sprintf("%024x", id)` or something similar